src/components/pages/rules/flyouts/ruleNew/ruleNew.scss

```diff
@@ -39,4 +39,10 @@
     @include rem-fallback(left, 180px);
     @include rem-fallback(top, 42px);
   }
+
+  .delete-btn {
+    @include rem-fallback(margin-left, 0px);
```

Comment:
It shouldn't be necessary for 0px to use rem-fallback.  
This should be sufficient:
margin-left: 0;

Reply:
fixed